### ISSUE TITLE
If used before or after wal -i (in a script or manually), animation freezes mid-transition

### ISSUE THREAD

**Issue by tka4nik** [LGFae/swww#362]
- Fedora 40
- Wayland Hyprland

I am trying to make a wallpaper-change bash script, and every time i try to pair two commands in any way (inside bash script, using -o of pywal, just pairing them through &&), wallpaper animation freezes mid-way (with random timing).

I don't know for certain if its an issue on my side or a bug sadly.

Example situation:

```
swww img $1 ///or the other way around
wal -q -i $1
```

```
swww img $1 && wal -q -i $1
```

```
wal -q -i /home/user/.dotfiles/wallpapers/wallpaper.png && swww img /home/user/.dotfiles/wallpapers/wallpaper.png
```

**Comment by LGFae** [comment]
Are you trying this on the latest git version or the latest release?

**Comment by tka4nik** [comment]
Yep, works on release 0.9.5, doesn't work on current master, thanks!

**Comment by LGFae** [comment]
This is really weird. I assume just setting the wallpaper without using wal just works?

**Comment by tka4nik** [comment]
Yeah, works fine both from shell script and manually if used without wal
